modules/private.php

```diff
@@ -281,15 +282,27 @@ static function add_private_dashboard_glance_items( $content ) {
 	 */
 	static function make_blog_private() {
 		update_option( 'blog_public', -1 );
-		// update_blog_public( 0, 0 );
 	}
 
 	/**
 	 * Changes a blog to public
 	 */
 	static function make_blog_public() {
 		update_option( 'blog_public', 1 );
-		// update_blog_public( 1, 1 );
+	}
+
+	/**
+	 * Returns the private page template for OPML.
+	 */
+	function hide_opml() {
+		global $current_user;
+
+		if ( $current_user && ( is_super_admin() || Jetpack_Private::is_private_blog_user( $wpdb->blogid, $current_user ) ) ) {
```

Comment:
Why not use `wp_get_current_user()` instead? Should we wrapp it inside a `! empty(`?

Reply:
Thanks for the input. I've made this change in all places that use the global variable `$current_user`